README.md

```diff
@@ -36,6 +36,11 @@ Popeye is available on Linux, OSX and Windows platforms.
    ```shell
    brew install derailed/popeye/popeye
    ```
+* Using `go install`
+ 
+ ```shell
+ go install github.com/derailed/popeye@latest
+ ```
```

Comment:
If it should follow the style of the other guides, the command should probably be indented
```suggestion
    ```shell
    go install github.com/derailed/popeye@latest
    ```
```